### ISSUE TITLE
FSCE finetuning results with CPE loss on COCO 

### ISSUE THREAD

**Issue by williamking5** [open-mmlab/mmfewshot#56]
Hi, thank you for great codes! I find that there is no finetuning results of FSCE with CPE loss in the model zoo page (contrastive loss is None).  Can you provide any training configurations with CPE loss? Thank you very much!
![image](https://user-images.githubusercontent.com/16954932/159247179-1bf31c84-cd3a-4bde-bf2b-e8374906a982.png)

**Comment by linyq17** [comment]
FSCE paper and released code both mention that the CPE loss does not work well in coco.  So we also do not provide the result on coco with CPE loss. To add CPE loss for coco, you can refer to the configuration of [voc](https://github.com/open-mmlab/mmfewshot/blob/main/configs/detection/fsce/voc/split1/fsce_r101_fpn_contrastive-loss_voc-split1_10shot-fine-tuning.py).

**Comment by williamking5** [comment]
Thank you for reply, I have tried to adapt PASCAL VOC configure for COCO contrastive loss but the mAP is much lower than the results reported in FSCE paper. If you have a plan to reproduce the results, we can discuss together.
